packages/wonder-blocks-banner/src/components/banner.js

```diff
@@ -88,20 +94,38 @@ type Props = {|
      * If present, dismiss button is on right side. If not, no button appears
      */
     onDismiss?: ?() => void,
+
+    /**
+     * The accessible label for the dismiss button.
+     * Please pass in a translated string.
+     */
+    dismissAriaLabel: string,
```

Comment:
**suggestion:** probably you have to specify this in the `argTypes.js` file.

Reply:
Oh, good catch! Thanks